Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Driver.hs

```diff
@@ -57,67 +57,67 @@ type API =
        )
 
 type DriverDocumentsInfoAPI =
-  ApiAuth 'DRIVER_OFFER_BPP 'READ_ACCESS 'DRIVERS
+  ApiAuth 'DRIVER_OFFER_BPP 'DRIVERS 'DOCUMENTS_INFO
```

Comment:
Seems like second argument `'DRIVER` is redundant here, because we can identify endpoint by third one only  `'DRIVERS 'DOCUMENTS_INFO`
But I'm ok with it if this code is clear enough.